client/my-sites/people/viewers-list/index.jsx

```diff
@@ -7,9 +7,9 @@ var React = require( 'react' ),
 /**
  * Internal dependencies
  */
-var PeopleListItem = require( 'my-sites/people/people-list-item' ),
+var PeopleListItem = require( 'my-sites/people/people-list-item' ).default,
 	Card = require( 'components/card' ),
-	PeopleListSectionHeader = require( 'my-sites/people/people-list-section-header' ),
+	PeopleListSectionHeader = require( 'my-sites/people/people-list-section-header' ).default,
```

Comment:
not yet unfortunately - at the moment, all have dependencies that have to be mocked before they can be safely loaded in a test